datalad/plugin/wtf.py

```diff
@@ -76,14 +76,24 @@ def __call__(dataset=None, sensitive=None, clipboard=None):
         else:
             cfg = None
 
+        from pkg_resources import iter_entry_points
         from datalad.ui import ui
         from datalad.api import metadata
-        from datalad.metadata import extractors as metaextractors
         from datalad.support.external_versions import external_versions
         import os
         import platform as pl
         import json
 
+        extractors={}
+        for ep in iter_entry_points('datalad.metadata.extractors'):
+            status = 'BROKEN'
+            try:
+                ep.load()
+                status = 'OK'
+            except:
+                pass
```

Comment:
would have been a good spot to use `exc_str` imho to not leave just noninformative BROKEN

Reply:
Done